content/docs/reference-events.md

```diff
@@ -31,44 +31,44 @@ number timeStamp
 string type
 ```
 
-> Note:
+> Catatan:
 >
-> As of v0.14, returning `false` from an event handler will no longer stop event propagation. Instead, `e.stopPropagation()` or `e.preventDefault()` should be triggered manually, as appropriate.
+> Sejak v0.14, mengembalikan nilai `false` dari _event handler_ tidak akan menghentikan propagasi _event_. Sebagai gantinya, `e.stopPropagation()` atau `e.preventDefault()` perlu dipanggil secara manual.
 
 ### Event Pooling {#event-pooling}
 
-The `SyntheticEvent` is pooled. This means that the `SyntheticEvent` object will be reused and all properties will be nullified after the event callback has been invoked.
-This is for performance reasons.
-As such, you cannot access the event in an asynchronous way.
+`SyntheticEvent` dikumpulkan. Hal ini berarti objek `SyntheticEvent` akan digunakan kembali dan isi objek akan dikosongkan setelah _callback_ untuk _event_ tersebut dipanggil.
+Hal ini dilakukan untuk mengoptimalkan kinerja.
+Dengan demikian, _event_ tersebut tidak dapat diakses secara asinkron.
 
 ```javascript
 function onClick(event) {
-  console.log(event); // => nullified object.
-  console.log(event.type); // => "click"
-  const eventType = event.type; // => "click"
+  console.log(event); // => objek yang dikosongkan.
+  console.log(event.type); // => "klik"
```

Comment:
Sepertinya karena yg dimaksud keluaran dari console.log(event.type) jadi sebaiknya tidak perlu di translate? What do you think? 😄

Reply:
i see, benar jg sii yg dimaksud itu output event.type ya :smile: